web/src/app/preview-profile/page.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button } from "@/components/ui/button";
+import Link from "next/link";
+import EmailPreviewProfile from "./components/email-profile";
+import NameAndOcupation from "./components/name-and-ocupation";
+import ProfileLevelPreview from "./components/profile-level";
+import VectorsUser from "./components/vectors-user.svg";
+
+export default async function PreviewProfile() {
```

Comment:
Também atualizar o nome do compononte. Vamos começar a adotar o sufixo Page pra denotar páginas. Provavelmente a gente vai precisar atualizar outros arquivos.

```suggestion
export default async function ProfilePage() {
```